requirements-dev.txt

```diff
@@ -0,0 +1,3 @@
+-r ./requirements.txt
```

Comment:
Would it be possible to place requirements-dev.txt file within the tests directory in some way? There are too many open files at the top level already.